HackIllinois/Controllers/FlowControllers/HILoginFlowController.swift

```diff
@@ -190,20 +240,16 @@ extension HILoginFlowController: HILoginSelectionViewControllerDelegate {
                                       withUserInfo info: String?) {
         switch selection {
         case .github:
-            print("URL::\(HIAuthService.githubLoginURL())")
-            loginSession = SFAuthenticationSession(url: HIAuthService.githubLoginURL(), callbackURLScheme: nil) { [weak self] (url, error) in
-
+            loginSession = SFAuthenticationSession(url: URL(string: "https://api.hackillinois.org/auth/github/?redirect_uri=https://test.hackillinois.org/auth/?isiOS=1")!, callbackURLScheme: "https://test.hackillinois.org/auth/?isiOS=1") { [weak self] (url, error) in
```

Comment:
Avoid raw string literals, this information does not logically make sense to reside in the viewcontroller